svoyak-game-scheduler/src/main/java/net/egork/telegram/svoyak/scheduler/ScheduleChat.java

```diff
@@ -113,9 +121,13 @@ public void processMessage(Message message) {
             break;
         case "/register":
         case "регистрация":
-            if (currentGame == null) {
-                sendMessage("Игра не начата");
-            } else if (currentGame.getPlayers().size() == currentGame.getMaxPlayers()) {
+            if (currentGame == null)
+                createGame();
+
+            if (currentGame == null)
```

Comment:
Please use brackets even for one liners
Please do not use excessive blank lines